tracing/tracing.go

```diff
@@ -0,0 +1,33 @@
+package tracing
+
+import (
+	propjaeger "go.opentelemetry.io/contrib/propagators/jaeger"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/trace/jaeger"
+	"go.opentelemetry.io/otel/propagation"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/trace"
+)
+
+// InitTracer creates an OTel TracerProvider that exports the traces to a Jaeger collector.
+func InitTracer(serviceName, collectorEndpoint string) (tp trace.TracerProvider, closer func(), err error) {
+	tp, closer, err = jaeger.NewExportPipeline(
```

Comment:
Do we want to have support for other exporters as well (other than Jaeger)?

Reply:
not for now (: